CMakeLists.txt

```diff
@@ -535,6 +535,10 @@ function(list_licenses OUTPUT MODULES)
     foreach(module ${_MODULES})
         set(license_path "../../third_party/${module}/LICENSE")
         get_filename_component(fullname "${license_path}" ABSOLUTE)
+        if(NOT EXISTS ${fullname})
```

Comment:
Instead of this special case, let's copy the `LICENSE.md` file and name it `LICENSE`.

Reply:
It's done.